wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -325,9 +335,7 @@ pub enum WalletCommand {
         margin_ratio_per_thousand: String,
 
         /// The key that can decommission the pool. It's recommended to keep the decommission key in a cold storage.
-        /// If not provided, the selected account in this wallet will control both decommission and staking.
-        /// This is NOT RECOMMENDED.
-        decommission_key: Option<HexEncoded<PublicKey>>,
+        decommission_key: HexEncoded<Destination>,
```

Comment:
This is wrong. This should not be a hex-encoded destination. This should be an address.

Reply:
Remember that destinations are `Addressable`.